loom_js_test/src/config.ts

```diff
@@ -36,6 +38,23 @@ export function getTestUrls() {
   }
 }
 
+export async function pollForBlockChange(
+  user: Entity,
+  currentBlock: BN,
+  maxIters: number,
+  sleepTime: number
+): Promise<BN> {
+  let blk = await user.getCurrentBlockAsync()
```

Comment:
This call is redundant, the loop will be entered regardless, and will execute the same call as long as initial value of `maxIters` is greater than zero (which it is in every use case so far).